Upstream: Yes (mailed)
Reason: Typo and specifying CRDA_PATH manually
Source: Daniel Mierswa <impulze@impulze.org>

From 14a0da690de4e8803809f5da970b42c953990f03 Mon Sep 17 00:00:00 2001
From: Daniel Mierswa <impulze@impulze.org>
Date: Thu, 5 Feb 2009 02:11:41 +0100
Subject: [PATCH] CDRA_PATH typo, also let user override CRDA_PATH

---
 Makefile             |    6 +++---
 debian-example/rules |    6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/Makefile b/Makefile
index a719f59..32626a1 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 # Install prefix
 PREFIX = /usr
-CDRA_PATH = $(PREFIX)/lib/crda
+CRDA_PATH ?= $(PREFIX)/lib/crda
 
 MANDIR ?= $(PREFIX)/share/man/
 
@@ -32,7 +32,7 @@ key.priv.pem:
 # Distributions wishing to just use John's database
 # can just call make install.
 install: regulatory.bin.5.gz
-	install -o 0 -g 0 -m 755 -d $(DESTDIR)/$(CDRA_PATH)
-	install -o 0 -g 0 -m 644 regulatory.bin $(DESTDIR)/$(CDRA_PATH)/regulatory.bin
+	install -o 0 -g 0 -m 755 -d $(DESTDIR)/$(CRDA_PATH)
+	install -o 0 -g 0 -m 644 regulatory.bin $(DESTDIR)/$(CRDA_PATH)/regulatory.bin
 	mkdir -p $(DESTDIR)/$(MANDIR)/man5/
 	install -m 644 -t $(DESTDIR)/$(MANDIR)/man5/ regulatory.bin.5.gz
diff --git a/debian-example/rules b/debian-example/rules
index dab9255..34a08a5 100755
--- a/debian-example/rules
+++ b/debian-example/rules
@@ -3,8 +3,8 @@
 include /usr/share/cdbs/1/rules/debhelper.mk
 
 PREFIX = /usr
-CDRA_LIB = $(PREFIX)/lib/crda
+CRDA_LIB ?= $(PREFIX)/lib/crda
 
 install/wireless-regdb::
-	install -o 0 -g 0 -m 755 -d debian/$(cdbs_curpkg)/$(CDRA_LIB)
-	install -o 0 -g 0 -m 644 regulatory.bin debian/$(cdbs_curpkg)/$(CDRA_LIB)/regulatory.bin
+	install -o 0 -g 0 -m 755 -d debian/$(cdbs_curpkg)/$(CRDA_LIB)
+	install -o 0 -g 0 -m 644 regulatory.bin debian/$(cdbs_curpkg)/$(CRDA_LIB)/regulatory.bin
-- 
1.6.1.2

